.gitignore

```diff
@@ -45,6 +45,9 @@
 !/bin/soft_oal.dll
 !/bin/xaudio2_9redist.dll
 
+# Test Programs
+!/bin/test/
```

Comment:
Hm why enforce ignoring its contents? shouldn't it allow .self/.elf file modifications?

Reply:
It's the opposite 🤔

Reply:
Carry on then.